.github/workflows/asset_dispatch.yml

```diff
@@ -16,11 +16,13 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
+      - name: Export node version
+        id: node_version
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
```

Comment:
Is this new syntax?  It's good!  But we had to use a workaround with environment variables in the past and still do in some workflows: https://github.com/sanger/crawler/blob/develop/.github/workflows/automated_release.yml#L61-L63